src/View/Widget/RadioWidget.php

```diff
@@ -126,7 +126,7 @@ public function render(array $data, ContextInterface $context): string
      * Disabled attribute detection.
      *
      * @param array<string, mixed> $radio Radio info.
-     * @param array|true|null $disabled The disabled values.
+     * @param array|true|string|null $disabled The disabled values.
```

Comment:
Interesting: The type has been added in 5.x, but the docblock part is missing:
https://github.com/cakephp/cakephp/blob/5.x/src/View/Widget/RadioWidget.php#L129

Reply:
Psalm complained without that `string` type because otherwise the 
```
!is_array($disabled)
```
at the end when returning wouldn't be needed. So I am curious why we don't have it in `5.x`

Reply:
We do, but not in docblock